oauth.go

```diff
@@ -438,11 +438,11 @@ func encodeUserParams(userParams map[string]string) string {
 
 // DEPRECATED: Use Post() instead.
 func (c *Consumer) PostForm(url string, userParams map[string]string, token *AccessToken) (resp *http.Response, err error) {
-	return c.Post(url, userParams, token)
+	return c.Post(url, "", userParams, token)
 }
 
-func (c *Consumer) Post(url string, userParams map[string]string, token *AccessToken) (resp *http.Response, err error) {
-	return c.makeAuthorizedRequest("POST", url, LOC_BODY, "", userParams, token)
+func (c *Consumer) Post(url string, body string, userParams map[string]string, token *AccessToken) (resp *http.Response, err error) {
```

Comment:
After some quick searching, it looks like some people are using the existing Post method.  So, adding a body parameter here would break them.  

On the one hand, we could ask them to update their code, but I'd prefer to avoid that.

How would you feel about creating a new method, maybe "PostWithBody" ... that way people using Post() can continue to use it without breaking, and you can use PostWithBody to achieve the functionality you need?